### PR TITLE
Update data-structure-on-rocksdb.md for Hyperloglog

### PR DESCRIPTION
**Pull request by tutububug**

### CODE REVIEW

File: community/data-structure-on-rocksdb.md
Comment (mapleFU) [inline community/data-structure-on-rocksdb.md:334]: Would you mind try Bitmap structure? "bitfield" command might help in this case?

File: community/data-structure-on-rocksdb.md
Comment (mapleFU) [inline community/data-structure-on-rocksdb.md:338]: we can denote the format of count. In current implementation, the count is an `count-to-string` ?

File: community/data-structure-on-rocksdb.md
Comment (tutububug) [reply community/data-structure-on-rocksdb.md:332]: I had changed my previous implementation of the storage format, currently, the format is the same as the bitmap's, and each HLL subkey represents a segment with 1024 registers. 
Also, I tried to use bitfield, but finally, I found that it requires byte alignment because of 'memcpy' (it should only be able to operate bits within a byte). The implementation of Redis only uses 6 bits, not aligned, to save more space, so it seems not suitable. 
However, another problem arises. When the register is set to 6-bit, unit test runs fail on a few OS releases in CI, but there is no problem when it is set to 8-bit, maybe it has something to do with byte alignment, but I don’t know the reason yet. Therefore, the current implementation still maintains 8 bits without using bitfield.
If I understand something wrong, correct me, thanks.

**Comment by mapleFU** [comment]
Thanks! I'll take a careful round these two days

File: community/data-structure-on-rocksdb.md
Comment (PragmaTwice) [inline community/data-structure-on-rocksdb.md:341]: So it's now not actually `register_index`? But something like segment index.

**Review by PragmaTwice — COMMENTED** [review]
Looks good to me.

It can be merged after the implementation is merged into unstable.

**Review by git-hulk — APPROVED** [review]

File: community/data-structure-on-rocksdb.md
Comment (mapleFU) [inline community/data-structure-on-rocksdb.md:49]: TODO: fix it to HyperLogLog

Comment (PragmaTwice) [reply community/data-structure-on-rocksdb.md:49]: It's already removed.

**Pull request merged**
